bin/runtime-common/src/integrity.rs

```diff
@@ -289,3 +289,32 @@ where
 	assert_bridge_messages_pallet_constants::<R, MI>(params.messages_pallet_constants);
 	assert_bridge_pallet_names::<B, R, GI, MI>(params.pallet_names);
 }
+
+pub fn check_message_lane_weights<C: Chain, T: frame_system::Config>(
+	extra_storage_proof_size: u32,
```

Comment:
Also maybe give some prefixes to arguments - like `bridged_` or `this_`. Because e.g. `extra_storage_proof_size` is for bridged chain and other args are about this chain. WDYT?

Reply:
Good point. Done.